cobc/cobc.c

```diff
@@ -8978,6 +8978,12 @@ main (int argc, char **argv)
 	cb_saveargc = argc;
 	cb_saveargv = argv;
 
+	/* This value is used when setting the format in FIXED mode in
+	   pplex, but it might happen that the value has not yet been 
+	   initialized in the configuration file.
+	*/
+	cb_config_text_column = 72;
```

Comment:
Just to double-check: this still happens before `cobc_set_source_format` is called, which sets it?

Reply:
In `pplex.y`, `set_source_format` performs:
```
	case CB_FORMAT_FIXED:
	case CB_FORMAT_AUTO:
	case CB_FORMAT_COBOL85:
		text_column = cb_config_text_column; /* 72 by default */
```
So, before this change, if a configuration file did not set the `text_column` parameter, the `format=fixed` parameter would set `text_column` to 0, instead of 72.

Reply:
I see, then the missing piece is to set `source_format` in the `struct list_files` after ppopen.

Reply:
Maybe we should create a ticket and postpone a complete fix for later, as this problem is unrelated to this PR (though I found the bug by testing it).

Reply:
I'll handle the issue creation and removing of `cb_config_text_column` completely (because it just does not work as it did, so better "break hard").